From 0fac4301918ee2591871705330bc937e0adb34a9 Mon Sep 17 00:00:00 2001
From: hdduy110697 <hdduy110697@gmail.com>
Date: Mon, 18 Nov 2019 20:12:25 +0700
Subject: [PATCH 4/6] applyPatches

---
 applyPatches.bat | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/applyPatches.bat b/applyPatches.bat
index 849c685..5e9d1de 100644
--- a/applyPatches.bat
+++ b/applyPatches.bat
@@ -18,4 +18,6 @@ cd patches
 call apply_patches_with_revert.bat
 popd
 
-:quit
\ No newline at end of file
+:quit
+
+echo nothing to do here
\ No newline at end of file
-- 
2.22.0.windows.1

